apps/web/app/(landing)/save-image/opengraph-image.tsx

```diff
@@ -0,0 +1,10 @@
+import { LANDING_PAGES_CONFIG } from '@/lib/landing-pages/landing-page-config'
+import { generateOGImage, runtime, alt, size, contentType } from '@/lib/landing-pages/og-image-generator'
+
+const config = LANDING_PAGES_CONFIG['save-image']!
+
+export { runtime, alt, size, contentType }
+
+export default async function Image() {
+  return generateOGImage(config)
```

Comment:
you can juste create [slug]/opengraph-image and get the `params` to get the right config instead of creating many files. Please update this